.github/workflows/merge.yml

```diff
@@ -410,6 +410,27 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 2
 
+      - name: Validate downloaded file after saved changes for Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          echo "upload the test document"
+          cargo run --bin safe --release -- --log-output-dest=data-dir files upload "./resources/test_doc.txt" --retry-strategy quick
+          echo "validate the test document contains the number 1 in its alphabetic data"          
+          $condition = Get-Content ./resources/test_doc.txt | ForEach-Object {$_ -Replace "[^0-9]", ""}
+          if ( $condition -ne 1) { exit 1 }
+          echo "add the values 2 and 3 to the test document"
+          "23" >> "./resources/test_doc.txt"
+          echo "Upload the recently changed test document"
+          cargo run --bin safe --release -- --log-output-dest=data-dir files upload "./resources/test_doc.txt" --retry-strategy quick
+          echo "Download the test document"
+          cargo run --bin safe --release -- --log-output-dest=data-dir files download --retry-strategy quick
+          echo "Test if the document includes the new values 2 and 3"
+          $condition = Get-Content C:\Users\runneradmin\Downloads\safe_files\test_doc.txt | ForEach-Object {$_ -Replace "[^0-9]", ""}
+          if ( $condition -ne 123) { exit 1 }
```

Comment:
I think it's worth digging into what this test is actually doing. 

`files upload` will upload a file, as chunks. 

If that file has changed, chunks will be different. 

You're not specifically testing that the file has changed on the network, because it doesn't. You're actually testing if the file reference was updated when you upload a new file with the same name I think. 

(@JasonPaulGithub Have a dig into what `files download` is doing here)

I'm not sure this test is a valid use test case given the above. 🤔

Reply:
Yes to this. In any case, if I upload a file (with same name and path, but new contents), and download grabs it all. I should get the latest file. That's what I would assume as a user, is this correct?